go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/web-platform-tests/wpt.fyi
 
-go 1.23.6
+go 1.23.7
```

Comment:
All the go version in Dockerfile need to be bumped as well, e.g. webapp/web/Dockerfile

Reply:
For future reference, there is a Golang upgrade guide here, https://github.com/web-platform-tests/wpt.fyi/blob/main/docs/upgrading-go.md

Reply:
> All the go version in Dockerfile need to be bumped as well, e.g. webapp/web/Dockerfile

OK, I think I got the top-level Dockerfile.  And, the search one had already been updated.